contacts/models.py

```diff
@@ -0,0 +1,28 @@
+from django.db import models
+from django.core.validators import RegexValidator
+
+
+class Contact(models.Model):
+    phone_regex = RegexValidator(
+            regex=r'^\+?1?\d{9,20}$',
+            message="Phone number must be entered in the format: '+999999999'. Up to 20 digits allowed."
+    )
+
+    name = models.CharField(max_length=50)
+    email = models.EmailField(max_length=30)
+    phone_number = models.CharField(validators=[phone_regex], max_length=20, blank=True)
+
+    def __str__(self):
+        return self.name
+
+
+class Address(models.Model):
+    address = models.ForeignKey(Contact, on_delete=models.CASCADE)
```

Comment:
1. the FK fieldname is supposed to be the referenced model name rather than the self.
reason: while using the reference in code, we would write `address.contact` rather than `address.address`.
let me know if it makes sense!

2. `on_delete=models.CASCADE` is set by-default in FK, we don't need to re-set it here!